### PR TITLE
add more style exceptions

### PR DESCRIPTION
**Pull request by martenson**
In order to allow https://github.com/galaxyproject/galaxy/pull/76 to proceed.

### CODE REVIEW

**Comment by dannon** [comment]
+1

**Comment by jmchilton** [comment]
-0 - the project standard should be `pep8` not [PEP-8](https://www.python.org/dev/peps/pep-0008/). The former is more useful, actionable, and represents the current prevailing community standard. But again - it is better to have and enforce a standard I dislike than not have a standard or not enforce it

File: setup.cfg [hunk 1]
```diff
@@ -1,4 +1,7 @@
 [flake8]
-# 201 and 202 are spaces after ( and before ), 501 is line length. These
-# are exceptions allowed (encouraged?) by Galaxy style guidelines.
-ignore = E201,E202,E501
+# These are exceptions allowed (encouraged?) by Galaxy style guidelines.
+# 128 continuation line under-indented for visual indent
+# 201 and 202 are spaces after ( and before )
+# 203 whitespace before ':'
```
Comment (natefoo) [inline setup.cfg:5]: Do we encourage this?

Comment (martenson) [inline setup.cfg:5]: I think we _allow_ it at least until the next linting team meeting where we decide on these issues.

Comment (jxtx) [inline setup.cfg:5]: > 1. Other lines should be formatted for readability, recognizing the existence of wide screens and scrollbars (sometimes a 200 character line is more readable, though rarely).

So I think yes we do for E501. Or at least, we encourage completely ignoring the rule in favor of "use whatever line length makes the code most readable".

E201 and E202 I think  more fall under tolerate.

Comment (dannon) [inline setup.cfg:5]: We've agreed that we break with E201, E202, and E501, and that everyone should follow the break with 201/202 the same way we do ( extra_space ).  E128 and E203 are a compromise to relax enforcement across the board, but there hasn't been an agreement to encourage anything other than 'make it readable'.

Comment (jmchilton) [inline setup.cfg:5]: @martenson If there is really a closed team meeting on coding style - I will leave the meeting. You guys can make a closed decision - but I don't want to have any part of it. These things should be discussed in the open.

Comment (martenson) [inline setup.cfg:5]: @jmchilton I have no problem with opening the hangout to public.

**Comment by carlfeberhard** [comment]
+1 - although I'd rather have the script be lax on (non-syntactic) whitespace and comments.

**Pull request merged**